### ISSUE TITLE
Improve Sidebar/Aside Design

### ISSUE THREAD

**Issue by MelSumner** [emberjs/website#3136]
Design a better sidebar for the site (pages that currently have sidebars are the guides, api, deprecations, and blog) 

- should be responsive/device agnostic
- should be accessible (A11y) 
- should complement the current site

**Comment by MelSumner** [comment]
Possible design: 
![image](https://user-images.githubusercontent.com/4587451/35594344-73834cd6-05d8-11e8-9312-a28322ebaa6f.png)

**Comment by jkumar712** [comment]
<img width="1440" alt="sidebar-1" src="https://user-images.githubusercontent.com/16778890/36475296-341659cc-1720-11e8-971e-f63b94e9ad81.png">

**Comment by MelSumner** [comment]
@locks @jenweber @wifelette any preferences? We're probably going to move this over to be an ember-styleguide component sooner rather than later.

**Comment by MelSumner** [comment]
Closing this issue since we settled on something we liked (currently visible in the guides).
